src/openrct2/windows/dropdown.c

```diff
@@ -174,8 +174,15 @@ void window_dropdown_show_text_custom_width(sint32 x, sint32 y, sint32 extray, u
     _dropdown_item_width = width;
     _dropdown_item_height = (flags & DROPDOWN_FLAG_CUSTOM_HEIGHT) ? custom_height : 10;
     gDropdownNumItems = (sint32)num_items;
-    _dropdown_num_columns = (gDropdownNumItems + DROPDOWN_TEXT_MAX_ROWS - 1) / DROPDOWN_TEXT_MAX_ROWS;
-    _dropdown_num_rows = (gDropdownNumItems + _dropdown_num_columns - 1) / _dropdown_num_columns;
+    // There must always be at least one column to prevent dividing by zero
+    if (gDropdownNumItems == 0) {
+        _dropdown_num_columns = 1;
+        _dropdown_num_rows = 0;
+    }
+    else {
```

Comment:
The coding style prescribes braces on their own line. I maintain these rules for new code.